src/api/app/views/webui/project/show.html.erb

```diff
@@ -53,7 +53,7 @@
                 <% if !@has_patchinfo %>
                     <li>
                       <%= sprite_tag 'exclamation' %>
-                      <%= link_to_if(User.current.can_modify?(@project), 'Patchinfo missing', { controller: 'patchinfo', action: 'new_patchinfo', project: @project, package: 'patchinfo' }, method: :post) %>
+                      <%= link_to_if(User.current.can_modify?(@project), 'Patchinfo missing', { controller: 'patchinfo', action: 'create', project: @project, package: 'patchinfo' }, method: :post) %>
```

Comment:
Could we use the new routes helper? :thinking:

Reply:
hey @DavidKang  which one?

Reply:
@vpereira this one `patchinfo_create_path`

Reply:
instead of using the hash, we could use the route helper `patchinfo_create_path`